Frontend/android-native/generate_apk.sh

```
@@ -0,0 +1,58 @@
+    cat merchant-properties/yatri.txt > local.properties
+elif [[ "$1" =~ .*"ys".* ]];
+then user="yatriSati"
+    partner="yatriSatiPartner"
+    echo "--------------------------yatriSati------------------------"
+    cat merchant-properties/yatriSati.txt > local.properties
+fi
+
+if [[ "$1" =~ .*"User".* ]];
+then 
+    echo " ---------- Customer prod:android :- --------------"
+    cd ../ui-customer
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)